<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/src_pic/QQ.jpg</normaloff>:/src_pic/QQ.jpg</iconset>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>-10</y>
     <width>201</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src_pic/message.png</normaloff>:/src_pic/message.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src_pic/setting.png</normaloff>:/src_pic/setting.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>71</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 153, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>781</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src_pic/qq_space.png</normaloff>:/src_pic/qq_space.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_group">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src_pic/friends.png</normaloff>:/src_pic/friends.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/src_pic/qq.svg);

border-width:0px;
border-radius:15px;
border-style:solid;
border-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_friend">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_friend_delete">
   <property name="geometry">
    <rect>
     <x>887</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-radius:13px;
border-style:solid;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_friend_invite">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-radius:13px;
border-style:solid;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>邀请加入群聊</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_group">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>330</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del_group_mem">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src_pic/insert.png</normaloff>:/src_pic/insert.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>listWidget</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>textEdit</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_group</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>label_friend</zorder>
  <zorder>pushButton_friend_delete</zorder>
  <zorder>pushButton_friend_invite</zorder>
  <zorder>listWidget_group</zorder>
  <zorder>pushButton_del_group_mem</zorder>
  <zorder>pushButton_insert</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
